### PR TITLE
Expose and use the ubiquity /source endpoint.

### PR DESCRIPTION
**Pull request by oSoMoN**

### CODE REVIEW

File: packages/subiquity_client/test/install-sources.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- id: ubuntu-desktop
+  default: true
+  name:
+    en: Ubuntu Desktop
+  description:
+    en: Standard Ubuntu Desktop image
+  path: ubuntu-desktop-minimal.ubuntu-desktop.squashfs
+  size: 10
+  type: fsimage-layered
+  variant: desktop
```
Comment (jpnurmi) [inline packages/subiquity_client/test/install-sources.yaml:19]: nit-pick: linebreak

File: packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_model.dart [hunk 2]
```diff
@@ -35,4 +40,18 @@ class UpdateOtherSoftwareModel extends ChangeNotifier {
...
+  Future<void> selectInstallationSource() {
+    String sourceId;
+    switch (installationMode) {
+      case InstallationMode.normal:
+        sourceId = 'ubuntu-desktop';
+        break;
+      case InstallationMode.minimal:
+        sourceId = 'ubuntu-desktop-minimal';
+        break;
+    }
```
Comment (jpnurmi) [inline packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_model.dart:54]: I like to make enum extensions for enum->string conversions - makes the code pretty :slightly_smiling_face:

**Review by jpnurmi — APPROVED** [review]
LGTM :+1:

I added a couple of comments but they are unimportant cosmetic things so feel free to ignore them.

File: packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_model.dart (hunk 2 above)
Comment (oSoMoN) [reply packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_model.dart]: Good suggestion, I did that.

**Review by jpnurmi — APPROVED** [review]

**Pull request merged**